src/components/PriceBar/CourtTemplate.tsx

```diff
@@ -0,0 +1,92 @@
+import { Box, Text, Flex, Icon } from "@chakra-ui/react";
+import React from "react";
+import AddCourt from "@/assets/svg/PriceBarSvg/addcourt.svg";
+import CourtIcon from "@/assets/svg/PriceBarSvg/courticon.svg";
+
+const CourtInfo: React.FC = () => {
+  const courts = [
+    { id: 1, name: "Example", icon: <CourtIcon /> },
+    { id: 2, name: "1", icon: <CourtIcon /> },
+  ];
+
+  const listCourts: JSX.Element[] = courts.map((court) => {
+    return (
+      <Box
+        as="button"
+        role="group"
+        w="full"
+        marginTop="1"
+        marginRight="4"
+        key={court.id}
+        data-testid="courtTemplate"
+      >
+        <Flex flexDirection="column" align="center" justify="flex-start">
+          <Icon
+            width="40px"
+            height="40px"
+            boxShadow="0.5px 1px 3px grey"
+            viewBox="0 0 20 36"
+            p="2px"
+            _groupHover={{ outline: "3px solid grey" }}
+            color="fontcolor.primary"
+            _groupActive={{ outline: "3px solid black" }}
+          >
+            {court.icon}
+          </Icon>
+          <Text
+            marginTop="1"
+            as="span"
+            color="background.primary"
+            fontSize="9px"
+            fontWeight="bold"
+            lineHeight="13px"
+          >
+            {court.name}
+          </Text>
+        </Flex>
+      </Box>
+    );
+  });
+
+  return <Box display="flex">{listCourts}</Box>;
+};
```

Comment:
It seems like you have 2 components nested with each other in one file. I would recommend to divide them into 2 separate files with passing props through.